src/gui/components/record/conflict/conflictDialog.tsx

```diff
@@ -0,0 +1,345 @@
+ * you may not use, this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing software
+ * distributed under the License is distributed on an "AS IS" BASIS
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND either express or implied.
+ * See, the License, for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * Filename: RecordTabBar.tsx
+ * Description:
+ *   TODO
+ */
+
+import * as React from 'react';
+import Button from '@mui/material/Button';
+import {styled} from '@mui/material/styles';
+import Dialog from '@mui/material/Dialog';
+import DialogTitle from '@mui/material/DialogTitle';
+import DialogContent from '@mui/material/DialogContent';
+import DialogContentText from '@mui/material/DialogContentText';
+import DialogActions from '@mui/material/DialogActions';
+import CloseIcon from '@mui/icons-material/Close';
+import Typography from '@mui/material/Typography';
+import IconButton from '@mui/material/IconButton';
+import InfoOutlinedIcon from '@mui/icons-material/InfoOutlined';
+import {CircularProgress} from '@mui/material';
+import {Grid, Box} from '@mui/material';
+
+export function ConflictButton(props: any) {
+  return (
+    <Button
+      aria-label={props.text}
+      onClick={() => props.onButtonClick(props.id)}
+      value={props.value}
+      id={props.id}
+      variant={props.ischoose ? 'outlined' : 'contained'}
+      style={{height: '56px'}}
+      disabled={props.disabled}
+    >
+      {props.text}
+    </Button>
+  );
+}
+
+const ConflictDialog = styled(Dialog)(({theme}) => ({
+  '& .MuiDialogContent-root': {
+    padding: theme.spacing(2),
+  },
+  '& .MuiDialogActions-root': {
+    padding: theme.spacing(1),
+  },
+}));
+
+export interface DialogTitleProps {
+  id: string;
+  children?: React.ReactNode;
+  onClose: () => void;
+}
+
+const ConflictDialogTitle = (props: DialogTitleProps) => {
+  const {children, onClose, ...other} = props;
+
+  return (
+    <DialogTitle sx={{m: 0, p: 2}} {...other}>
+      {children}
+      {onClose ? (
+        <IconButton
+          aria-label="close"
+          onClick={onClose}
+          sx={{
+            position: 'absolute',
+            right: 8,
+            top: 8,
+            color: theme => theme.palette.grey[500],
+          }}
+        >
+          <CloseIcon />
+        </IconButton>
+      ) : null}
+    </DialogTitle>
+  );
+};
+
+function ConflictDialogContent() {
+  return (
+    <Box pb={20} pt={10} pl={3} pr={3}>
+      <Grid container spacing={{xs: 2, md: 3}}>
+        <Grid
+          item
+          xs={12}
+          container
+          justifyContent="center"
+          alignItems="flex-end"
+        >
+          <Typography>
+            The most recent conflict has been pre-selected. Choose another using
```

Comment:
Needs ending `.`. Also, I have no idea what this is trying to communicate. Where does this appear on @lmannering's wireframes?